cfgov/unprocessed/apps/ccdb-landing-map/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "ccdb-landing-map",
+  "description": "The landing page components for consumer complaints",
+  "homepage": "https://www.consumerfinance.gov/data-research/consumer-complaints/",
+  "author": {
+    "name": "Consumer Financial Protection Bureau",
+    "email": "tech@cfpb.gov",
+    "url": "https://cfpb.github.io/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cfpb/cfgov-refresh.git"
+  },
+  "license": "SEE LICENSE IN TERMS.md",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "dependencies": {
+    "@cfpb/cfpb-core": "0.0.9",
+    "@cfpb/cfpb-layout": "0.0.9",
```

Comment:
These should be removable, as they are available in the wider project already. I removed them, reran `frontend.sh`, and everything still works for me locally.
```suggestion
```